doc/nrf/gs_toolchain.rst

```diff
@@ -0,0 +1,19 @@
+.. _gs_toolchain:
+
+Installing the |NCS| from a zip (experimental)
+##############################################
+
+.. warning::
+   Installing from a zip is not officially supported yet.
+   It should work fine for common use cases, but not all scenarios have been tested.
+
+To simplify the |NCS| installation on Windows operating systems, you can download a zip file that contains the full |NCS| toolchain.
+
+The toolchain zip includes the full `GNU Arm Embedded Toolchain`_, the Nordic Edition of |SES|, and all tools that are required to build and program |NCS| applications.
```

Comment:
I would list contents as bullet points. Also, if we have a link to "all tools that are required", I would include it here.